portal/templates/profile/profile_macros.html

```diff
@@ -240,29 +240,36 @@ <h4 id="patientDetailLoc" class="profile-item-title index-item">
         <div class="help-block with-errors" id="erroremail"></div>
     </div>
 {%- endmacro %}
+{% macro render_emailSelector(prefix='registration', status_title='Registration Status:', label='not registered') -%}
+    <div id="{{prefix}}StatusContainer" class="{{prefix}}-status-container">
+        <span class="text-muted">{{_(status_title)}}</span>
```

Comment:
I think you'll need to move the `_()` call to the macro definition line (see below) for those strings to be properly extracted
`{% macro render_emailSelector(prefix='registration', status_title=_('Registration Status'), label=_('not registered')) -%}`